tests/Microsoft.Bot.Builder.Tests/ShowTyping_MiddlewareTests.cs

```diff
@@ -69,12 +91,35 @@ public async Task ShowTyping_TestMiddleware_ImmediateResponse_5SecondInterval()
                 .StartTestAsync();
         }
 
+        [Fact]
+        public async Task ShowTyping_TestMiddleware_ImmediateResponse_When_Running_As_Skill()
```

Comment:
Consider combining this test with ShowTyping_TestMiddleware_1_Second_Interval and using a Theory instead of Fact to avoid code duplicaiton.